src/components/CreateRequest/Details/index.js

```diff
@@ -150,6 +151,15 @@ class Details extends Component {
       return;
     }
 
+    const documentURI = this.state.documentURI;
+    const requestTrainingDSURI = this.state.requestTrainingDS;
+
+    const isValidDocumentURI = snetValidator({ documentURI }, { documentURI: { url: true } });
+    const isValidRequestTrainingDSURI = snetValidator(
+      { requestTrainingDSURI },
+      { requestTrainingDSURI: { url: true } }
+    );
```

Comment:
use single validationConstraints for this form.

Reply:
We have the Blockchain Validator help function for all BN Validations. We will look on how we can integrate both as it has lots of dependency on Web3 objects (Utility).